src/uct/ib/ud/base/ud_ep.c

```diff
@@ -561,9 +569,19 @@ static ucs_status_t uct_ud_ep_create(uct_ud_iface_t *iface, int path_index,
     *new_ep_p            = ucs_derived_of(new_ep_h, uct_ud_ep_t);
     (*new_ep_p)->conn_sn = conn_sn;
 
+    /* Adjust flags, since the endpoint is created as full duplex and p2p by
```

Comment:
maybe the endpoint should be created without any flags by default?
or use a lower-level function here (not uct_ep_create), so no need to add and then remove flags